qemu/tests/cfg/mmu_basic.cfg

```diff
@@ -0,0 +1,18 @@
+- mmu_basic:
+    type = mmu_basic
+    only pseries
+    only Host_RHEL.m8
+    no RHEL.6
+    virt_test_type = qemu
+    kill_vm = yes
+    cpu_info = "cat /proc/cpuinfo"
+    kernel_extra_params_remove = disable_radix
+    image_snapshot = yes
+    mmu_option = yes
+    variants:
+        - @default:
+            only RHEL.8
+        - hpt:
+            machine_type_extra_params = "max-cpu-compat=power8"
+            RHEL.7:
```

Comment:
For RHEL7 guest, if we can not get the MMU via `cpuinfo`, I don't think we need to  test it on this scenario. But maybe there are other ways to get it.
@MiriamDeng Could you help to confirm this issue and/or provide the useful command?

Reply:
The reply discussed with @MiriamDeng is that RHEL7 guest,  is a negative test.

Reply:
This test has been updated in the polarion case 186724.

Reply:
In fact, this is not strictly a negative test, if RHEL7 can get the MMU from cpuinfo in the future, we still need to support it.
Check the MMU exists and ignore if this is a RHEL7 guest will make the code stronger. Like: check MMU -> if empty -> is RHEL7 -> ignore.

Reply:
It;s ok for me.
correct